internal/prebuiltconfigs/tools/alloydb-postgres-admin.yaml

```diff
@@ -115,9 +115,62 @@ tools:
         type: string
         description: "The full resource name of the primary cluster for a SECONDARY instance. Required only if instanceType is SECONDARY. Otherwise don't ask"
         default: ""
+  alloydb-list-clusters:
+      kind: http
+      source: alloydb-api-source
+      method: GET
+      path: /v1/projects/{{.projectId}}/locations/{{.locationId}}/clusters
+      description: "Lists all AlloyDB clusters in a given project and location."
+      pathParams:
+        - name: projectId
+          type: string
+          description: "The GCP project ID to list clusters for."
+        - name: locationId
+          type: string
+          description: "The location to list clusters in (e.g., 'us-central1'). Use '-' to list clusters across all locations."
```

Comment:
Have we tested if the agents are good at setting the region/location ie do we need more examples in the tool description to reduce any issues with getting this correct?